tests/unit/test_handlers.py

```diff
@@ -1944,3 +1917,80 @@ def test_document_response_params_without_expires(document_expires_mocks):
     mocks['section'].get_section.assert_not_called()
     mocks['param_section'].add_new_section.assert_not_called()
     mocks['doc_section'].write.assert_not_called()
+
+
+@pytest.fixture()
+def operation_model_for_200_error():
+    operation_model = mock.Mock()
+    operation_model.has_streaming_output = False
+    return operation_model
+
+
+@pytest.fixture()
+def response_dict_for_200_error():
+    return {
+        'status_code': 200,
+        'body': (
+            b"<Error>"
+            b"<Code>SlowDown</Code>"
+            b"<Message>Please reduce your request rate.</Message>"
+            b"</Error>"
+        ),
+    }
+
+
+def test_500_status_code_set_for_200_response(
+    operation_model_for_200_error, response_dict_for_200_error
+):
+    handlers._handle_200_error(
```

Comment:
Ideally we aren't invoking private functions outside of their respective modules. Do we have specific cases we're testing in these unit tests that aren't already verified with the functional tests?